wolfcrypt/src/asn.c

```diff
@@ -11250,6 +11250,80 @@ static int wc_SetCert_LoadDer(Cert* cert, const byte* der, word32 derSz)
     return ret;
 }
 
+/* AltName structure contains: 
+   Sequence of OID + Octet string containing sequence of names.
+   WC_ALTNAMESOFFSET is the max header size: 2 sequences, one
+   octet string, and the OID */
+#define WC_ALTNAMESOFFSET (MAX_SEQ_SZ + WC_SUBJECTALTNAME_OID_SIZE \
+	                       + MAX_OCTET_STR_SZ + MAX_SEQ_SZ)
+
+int wc_CertAddAltName_ex(Cert *cert, byte *name, int nameSz, 
+						 cert_altname_type type) {
+	/* Append encoded value from name into cert->altNames */
+	byte *buf;
+	int i;
+	
```

Comment:
Please validate the parameters `cert` and `name`

Reply:
input implemented